examples/WIP_bickley_jet_cmc_cg.jl

```diff
@@ -0,0 +1,117 @@
+# set up boilerplate
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
+
+using ClimaCore.Geometry, LinearAlgebra, UnPack
+import ClimaCore: Fields, Domains, Topologies, Meshes, Spaces
+import ClimaCore.Operators
+import ClimaCore.Geometry
+using LinearAlgebra, IntervalSets
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+global_logger(TerminalLogger())
+
+# set up parameters
+const parameters = (
+    ϵ = 0.1,   # perturbation size for initial condition
+    l = 0.5,   # Gaussian width
+    k = 0.5,   # Sinusoidal wavenumber
+    ρ₀ = 1.0,  # reference density
+    c = 2,
+    g = 10,
+    D₄ = 1e-4, # hyperdiffusion coefficient
+)
+
+# set up grid
+n1, n2 = 16, 16
+Nq = 4
+domain = Domains.RectangleDomain(
+    -2π..2π,
+    -2π..2π,
+    x1periodic = true,
+    x2periodic = true,
+)
+mesh = Meshes.EquispacedRectangleMesh(domain, n1, n2)
+grid_topology = Topologies.GridTopology(mesh)
+quad = Spaces.Quadratures.GLL{Nq}()
+space = Spaces.SpectralElementSpace2D(grid_topology, quad)
```

Comment:
It is not clear to me what are the differences between mesh, grid_topology, quad, and space. I am not sure if we'd keep them all in the runscript. There should be a way to construct a struct and synthesize all these info.